experimenter/experimenter/nimbus-ui/src/components/PageEditBranches/FormBranches/index.tsx

```diff
@@ -227,6 +232,7 @@ export const FormBranches = ({
         className="my-3"
         noValidate
         onSubmit={handleSave}
+        validated={isValid && formSubmitted}
```

Comment:
there is something wrong, I don't think we are using states to show green checks, I would suggest doing some research more on this without state, also if you will see on other page if we change let's say expected no. of clients and if its valid it only shows green on that part, but in your case, all the other checkboxes are becoming green too, it shouldn't be doing that
![image](https://user-images.githubusercontent.com/25848231/227315391-3a4468c1-1e72-4e73-8c71-ecdde8fa2ebd.png)

Reply:
@yashikakhurana I will look into that! However, just to clarify, it seemed like the branches page already had the green checks implemented when you change an input field. As seen in the image you attached, I also attached an image from the branches page to simulate that. From what I could tell, the only difference between the audience and the branches page was that when the "Save" button was pressed, the audience page saved everything on the form. However, the branches page did not. Please correct me if I am wrong :) 
<img width="1175" alt="Screen Shot 2023-03-22 at 11 25 31 AM" src="https://user-images.githubusercontent.com/117552772/227320363-715981cb-275a-4c03-b1a7-c94f288adede.png">

https://user-images.githubusercontent.com/117552772/227320456-e3021d7b-480a-4eef-9c9a-bc5d82ec53b5.mov

Reply:
@yashikakhurana Thanks for catching that! I believe it should be updated now :) Thanks again.